littlelines/otwarchive
# Ao3 5885 restict user creation

# Pull Request Checklist


## Issue

https://otwarchive.atlassian.net/browse/AO3-885

## Purpose

What does this PR do?
Restricts admins ability to delete, hide, or modify user creations without correct roles.

## Testing Instructions
Check pages of work, external work, bookmark, and series to make sure the appropriate buttons for removing and marking as spam are not visible to admins without the correct roles.
Visit the links specified in the ticket to make sure those pages redirect for admins without correct roles, as well.

## References

Are there any other relevant issues / PRs / mailing lists discussions related to this?

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*

(If you have an account on JIRA, please include the same name in the "Full name" field on your JIRA profile,
so we can find you and assign you the issues you're working on.)

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

(If you do not fill this in, we will use your GitHub account name and will use "they" to avoid making assumptions about your gender.)
